code-challenges.test.js

```diff
@@ -14,38 +14,143 @@
+    return `${firstName} ${lastName} is ${value.occupation}.`
+  })
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       3 passed, 3 total
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+describe("numbersOnly", () => {
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+    const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+    const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+    expect(numbersOnly(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(numbersOnly(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
 
+// ReferenceError: numbersOnly is not defined
 
-// b) Create the function that makes the test pass.
+// b) Create the function that makes the test pass. 
+
+  //----Puesdocode---//
+    // declare a function called numbersOnly
+    // set a param of array
+    // declare a variable newArray
+        // use .filter to create and array of numbers using typeof
+    // use .map on newArray to return an array with the remainder of the values divided by 3 
 
+    const numbersOnly = (array) => {
+      let newArray = array.filter (value => 
+        typeof value === "number");
+        return newArray.map (value => {
+          return value % 3})
+    }
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
-
+describe("cubedSum", () => {
+  it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    const cubeAndSum1 = [2, 3, 4]
+    const cubeAndSum2 = [0, 5, 10]
+    expect(cubedSum(cubeAndSum1)).toEqual(99)
+    expect(cubedSum(cubeAndSum2)).toEqual(1125)
+  })
+})
 
 // b) Create the function that makes the test pass.
+
+    //----Puesdocode----//
+      // declare a function called cubedSum
+      // set param of array
+      // declare a variable called newArray
+        //use .map to return an array with the values cubed 
+      // use .reduce on newArray, with arguments of sum and value, to return to sum of the newArray
+
+  const cubedSum = (array) => {
+    let newArray = array.map(value => value**3)
+    return newArray.reduce((sum, value) => {
+      return sum + value
+    }, 0)
+  }
+
+  
+  // Test Suites: 1 passed, 1 total
+  // Tests:       2 passed, 2 total
```

Comment:
Awesome way to finish out the coding challenges!  You have great clean code. Really impressive!